### ISSUE TITLE
0.12.0pre2 wait_tap 'command' fails

### ISSUE THREAD

**Issue by shyukri** [calabash/calabash-ios#653]
```
irb(main):002:0> wait_tap("button {accessibilityLabel ENDSWITH 'nav more'}")
NameError: undefined local variable or method `command' for #<Calabash::Cucumber::InstrumentsActions:0x00000106882528>
	from /Users/mac_mini/.rvm/rubies/ruby-2.1.0/lib/ruby/gems/2.1.0/gems/calabash-cucumber-0.12.0.pre2/lib/calabash-cucumber/uia.rb:70:in `uia_wait_tap'
	from /Users/mac_mini/.rvm/rubies/ruby-2.1.0/lib/ruby/gems/2.1.0/gems/calabash-cucumber-0.12.0.pre2/lib/calabash-cucumber/actions/instruments_actions.rb:20:in `wait_tap'
	from /Users/mac_mini/.rvm/rubies/ruby-2.1.0/lib/ruby/gems/2.1.0/gems/calabash-cucumber-0.12.0.pre2/lib/calabash-cucumber/core.rb:212:in `wait_tap'
	from (irb):2
	from /Users/mac_mini/.rvm/rubies/ruby-2.1.0/bin/irb:11:in `<main>'

irb(main):004:0> touch("button {accessibilityLabel ENDSWITH 'nav more'}")
[
  [0] {
    "id"                   => nil,
    "selected"             => false,
    "visible"              => 1,
    "label"                => "bottom nav more",
    "frame"                => {
      "y"      => 0,
      "width"  => 64,
      "x"      => 0,
      "height" => 44
    },
    "enabled"              => true,
    "description"          => "<UIButton: 0x7b9c6c80; frame = (0 0; 64 44); alpha = 0.5; opaque = NO; autoresize = W; layer = <CALayer: 0x7b9c6d80>>",
    "alpha"                => 0.5,
    "value"                => nil,
    "rect"                 => {
      "center_x" => 288,
      "y"        => 524,
      "width"    => 64,
      "x"        => 256,
      "center_y" => 546,
      "height"   => 44
    },
    "accessibilityElement" => 1,
    "class"                => "UIButton"
  }
]

  "simulator_device"            => "iPhone",
  "version"                     => "0.12.0.pre1",
```

**Comment by towercap** [comment]
I've come across the exact same issue.

xcode installation location
/Applications/Xcode.app/Contents/Developer

xcode version
Xcode 6.1
Build version 6A1052d

calabash version
0.12.0

calabash.framework version
0.12.0

**Comment by Goginenni** [comment]
@krukow Fix this issue in https://github.com/calabash/calabash-ios/pull/666. Please have a look. Thanks.

**Comment by sasjo** [comment]
I have this issue with v 0.12.0

```
wait_tap "view:'UITabBarButton' marked:'Menu'"
NameError: undefined local variable or method `command' for #<Calabash::Cucumber::InstrumentsActions:0x007fe419187f68>
	from /Library/Ruby/Gems/2.0.0/gems/calabash-cucumber-0.12.0/lib/calabash-cucumber/uia.rb:70:in `uia_wait_tap'
	from /Library/Ruby/Gems/2.0.0/gems/calabash-cucumber-0.12.0/lib/calabash-cucumber/actions/instruments_actions.rb:20:in `wait_tap'
	from /Library/Ruby/Gems/2.0.0/gems/calabash-cucumber-0.12.0/lib/calabash-cucumber/core.rb:215:in `wait_tap'
	from (irb):5
	from /usr/bin/irb:12:in `<main>'
```

The workaround is to use this boilerplate version...

```
q = "view:'UITabBarButton' marked:'Menu'"
wait_for_element_exists q
wait_for_none_animating
touch q
````

**Comment by Goginenni** [comment]
Yes, this issue is still reproducible in 0.12.0 . the fix is merged today. It should be available in the next release.

**Comment by jmoody** [comment]
Fix will be available in 0.12.1.
